addons/common/functions/fnc_getCountOfItem.sqf

```diff
@@ -0,0 +1,31 @@
+#include "script_component.hpp"
+/*
+ * Author: Dedmen
+ * Return how many items of type _itemType the player has in his containers (Uniform, Vest, Backpack)
+ * Doesn't count assignedItems, weapons, weapon attachments, magazines in weapons
+ *
+ * Arguments:
+ * 0: unit <OBJECT>
```

Comment:
`Unit`

Reply:
Thought the same actually. But I just copied it from another function :D